src/node_report_module.cc

```diff
@@ -84,6 +84,19 @@ static void SetCompact(const FunctionCallbackInfo<Value>& info) {
   per_process::cli_options->report_compact = compact;
 }
 
+static void GetNetworkDisabled(const FunctionCallbackInfo<Value>& info) {
+  Environment* env = Environment::GetCurrent(info);
+  info.GetReturnValue().Set(env->options()->report_network_disabled);
+}
+
+static void SetNetworkDisabled(const FunctionCallbackInfo<Value>& info) {
+  Environment* env = Environment::GetCurrent(info);
+  CHECK(info[0]->IsBoolean());
+  Isolate* isolate = env->isolate();
+  env->options()->report_network_disabled
+    = info[0]->ToBoolean(isolate)->Value();
+}
+
```

Comment:
I feel like maybe I'm doing this part incorrectly, and it is what is causing the seg fault (I still can't get a core dump to verify unfortunately). 

WDYT @joyeecheung & @anonrig ?